src/main/web/templates/handlebars/content/t6-3.handlebars

```diff
@@ -15,63 +15,66 @@
                 <p>{{downloads.size}} download{{#if_ne downloads.size 1}}s{{/if_ne}}</p>
             </div>
         </div>
+        <div class="col-wrap">
         {{!-- LIST OF DOWNLOADS --}}
-        <div class="col col--md-36 col--lg-39">
-            <ul class="list--neutral margin-top--0">
-                {{#if downloads}}
-                    <h2 class="margin-top--2">{{labels.downloads}}</h2>
-                    {{#each downloads}}
-                        <li>
-                            <h3><a href='/file?uri={{concat uri "/" (fn file)}}'>{{title}}</a> ({{fs (concat uri "/" (fn file))}}, {{fe (concat uri "/" (fn file))}})</h3>
-                            <p class="flush">{{fileDescription}}</p>
-                        </li>
+        <div class="col col--lg-two-thirds margin-left--0">
```

Comment:
I wonder if we maybe want to update the margin to be `margin-left-lg--1` as on desktop viewports, the Downloads section doesn't align with the rest of the page content:

![image](https://user-images.githubusercontent.com/23668262/88370024-27ca3b80-cd89-11ea-9bbb-edb6aefb4e10.png)

When adding that utility class:

![image](https://user-images.githubusercontent.com/23668262/88370063-39134800-cd89-11ea-9c2d-87a6774f53cb.png)

If we use `margin-left-lg--1` it should also mean that the alignment in other viewports remains unaffected

Reply:
Good spot. Now resolved